Libplanet/Net/Swarm.cs

```diff
@@ -1379,7 +1385,7 @@ CancellationToken cancellationToken
                         {
                             _logger.Error(
                                 e,
-                                "FillBlockAsync() failed. retrying..."
+                                $"{nameof(FillBlocksAsync)}() failed: {e}\nRetrying..."
```

Comment:
As usually `e` takes lines long, this would be more readable:

```suggestion
                                $"{nameof(FillBlocksAsync)}() failed; retrying...: {e}"
```